src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -4168,6 +4168,17 @@ static void writePitch(XmlWriter& xml, const Note* const note, const bool useDru
     if (!alter && alter2) {
         xml.tag("alter", alter2);
     }
+
+    // if alter (from tonal pitch class) and alter2 (from note accidental) are empty we check
+    // if the note has a tuning to add the "alter" tag with tuning value / 100
+    if (!alter && !alter2) {
+        double tuning = note->tuning();
+        static const int INVALID_LINE = 100;
```

Comment:
unused variable, leads to a compiler warning

Reply:
My fault. I have updated it